### DISCUSSION TITLE
0021 XLS-21d: Allocating Asset Code Prefixes

### DISCUSSION THREAD

**Discussion by mDuo13** [XRPLF/XRPL-Standards#53] [XLS Proposals]
## The earlier version can be found by looking at the edit history.

**Comment by mvadari** [comment]
@mDuo13 should this spec be moved into PR and kept in the repo or should it be closed? Note that this does not mean that the spec has been accepted/implemented/finalized.

I will move it to PR by copy-pasting the existing draft if there is no response in the next week.
